addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,17 +2,15 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl">
-        <div t-on-mouseenter="this.props.freeze" t-on-mouseleave="this.props.refresh" t-attf-class="o_notification {{props.className}} border border-{{props.type}} bg-view mb-2 position-relative"
+        <div t-on-mouseenter="this.props.freeze" t-on-mouseleave="this.props.refresh" t-attf-class="o_notification {{props.className}} bg-view border border-{{props.type}} mb-2 p-3 position-relative rounded shadow-lg"
```

Comment:
convention: background utility classes after border ones, and rounded immediately after border.
note that we should theoretically adapt the preexisting code by moving `position-relative` at the beginning... but let's not do that in order to keep the diff slim.

```suggestion
        <div t-on-mouseenter="this.props.freeze" t-on-mouseleave="this.props.refresh" t-attf-class="o_notification {{props.className}} border border-{{props.type}} rounded bg-view mb-2 p-3 position-relative shadow-lg"
```